package.json

```diff
@@ -84,6 +86,7 @@
         "just-camel-case": "^4.0.2",
         "morphdom": "^2.6.1",
         "mustache": "^4.2.0",
+        "post-robot": "^8.0.31",
```

Comment:
Couldn't we use our post-robot, instead?

Reply:
I didn't get this @Deepak-Kharah is there any promise based messaging implementation of ours?

Reply:
Here's the adv post message: https://github.com/contentstack/adv-post-message

Reply:
@Deepak-Kharah we can have a separate ticket for it.

Reply:
Sure, let's create a ticket and attach it here for reference.